.pre-commit-config.yaml

```
@@ -119,18 +119,15 @@ repos:
           ?^ci/.*/.*\.cmake$|
           ?^cpp/.*/.*\.cmake\.in$|
           ?^cpp/.*/.*\.cmake$|
-          ?^cpp/.*/CMakeLists\.txt$|
-          ?^go/.*/CMakeLists\.txt$|
-          ?^java/.*/CMakeLists\.txt$|
-          ?^matlab/.*/CMakeLists\.txt$|
-          ?^python/.*/CMakeLists\.txt$|
+          ?.*CMakeLists\.txt$|
           )
         exclude: >-
           (
           ?^cpp/cmake_modules/FindNumPy\.cmake$|
           ?^cpp/cmake_modules/FindPythonLibsNew\.cmake$|
           ?^cpp/cmake_modules/UseCython\.cmake$|
           ?^cpp/src/arrow/util/config\.h\.cmake$|
+          ?^ci/conan/all/.*CMakeLists\.txt$|
```

Could you sort this list in alphabetical order?

```suggestion
          ?^ci/conan/all/.*CMakeLists\.txt$|
          ?^cpp/cmake_modules/FindNumPy\.cmake$|
          ?^cpp/cmake_modules/FindPythonLibsNew\.cmake$|
          ?^cpp/cmake_modules/UseCython\.cmake$|
          ?^cpp/src/arrow/util/config\.h\.cmake$|
```